src/ShallowWrapper.js

```diff
@@ -26,6 +26,7 @@ import {
   createShallowRenderer,
   renderToStaticMarkup,
 } from './react-compat';
+import SyntheticEvent from 'react/lib/SyntheticEvent';
```

Comment:
This should likely be re-exported from `react-compat`, not directly imported from `react/lib`.